conda-store-server/conda_store_server/api.py

```diff
@@ -290,20 +290,6 @@ def get_build_packages(
     )
 
 
-def get_build_lockfile(db, build_id: int):
```

Comment:
Removing generating lockfiles from database since we now use conda-lock's output.